test/DynamoCoreTests/PackageDependencyTests.cs

```diff
@@ -94,7 +94,7 @@ public void ZeroTouchPackageDependencyIsCollectedAndSerialized()
             Assert.AreEqual(package.Nodes.Select(n => n.ToString("N")), nodes);
         }
 
-        [Test,Category("FailureNET6")]
+        [Test,Category("WindowsOnly")]
```

Comment:
@marbelarrietaGlobant - why have you changed these categories?

Reply:
According to the acceptance criteria of the US, the tests that failed in net60 were reviewed and when they were run it was found that they are not failing anymore in net60 for windows version. Linux continues failing.

Reply:
@marbelarrietaGlobant I'd prefer if the changes in this PR are localized to just excel tests. 
Please leave these others as is - or add `WindowsOnly`, but keep the existing FailureNet6 in addition. Thanks.

Reply:
@mjkkirschner Ok I changed to FailureNET6 again to test in the future for Windows and Linux.